docs/docs/services/qix-engine/access-control.md

```diff
@@ -72,36 +84,38 @@ docker run -v <host folder path>:<container folder path> qlikcore/engine:<versio
     -S SystemDenyRulePath=/<container folder path>/deny.txt
 ```
 
-### More Code Examples
+For more examples using ABAC, see the
+[core-authorization](https://github.com/qlik-oss/core-authorization) repository,
+which contains running code examples on how to enable ABAC and how to provide rules to Qlik Associative Engine.
 
-The [core-authorization](https://github.com/qlik-oss/core-authorization) repository contains running code examples on
-how to enable ABAC and how to provide rules to Qlik Associative Engine.
+## Rules language
 
-## Rules Language
+The following section describes the syntax and semantics of the Qlik Associative Engine rules language.
```

Comment:
To me, writing "The following section ..." implies referring to the section **Rules language concepts**, when it is really all the following sub sections that cover this. Does it not make more sense to write "The following _sections_ ..."?

Reply:
Like above...